services/121-service/src/program-financial-service-provider-configurations/interfaces/required-username-password.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface RequiredUsernamePasswordInterface {
```

Comment:
For one the properties are optional and for the other they are not. 

The reason:
- I use the UsernamePasswordInterface in the ProgramFinancialServiceProviderConfigurationRepository with unrequired properties, because I do not want to throw a HttpException in a hidden place in the repository file. So the return is optional
- The non optional one I use in CBE service to pass around service after I throw a 400 eror in case the password is undefined / or create a failed transaction